content/en/docs/ambient/architecture/index.md

```diff
@@ -33,9 +84,8 @@ However, depending on the destination's capabilities, different behavior will oc
 If the destination is also captured, or otherwise has Istio proxy capabilities (such as a sidecar), the request will be upgraded to an encrypted {{< gloss "HBONE" >}}HBONE tunnel{{< /gloss >}}.
```

Comment:
I think we should swap HBONE with TLS here, we can have a separate doc on write protocol, TLS & HBONE but I don't think we're trying to sell 'HBONE" here

Reply:
Defer to this change as we currently have a glossary for hbone but nothing for TLS.

Reply:
Could update when https://github.com/istio/istio.io/pull/15004/files is merged from @bleggett